src/services/firebase/firestore.ts

```diff
@@ -221,18 +221,18 @@ export class FirestoreServices {
   };
 
   sendMessageWithFile = async (message: SendMessageProps) => {
-    const { path, extension, type } = message;
+    const { path, extension, type, fileName, size } = message;
 
-    if (!path || !extension || this.conversationId === null) {
-      console.error('Please provide path and extension');
-      return;
+    if (!path || !extension || this.conversationId === null || !type) {
+      throw new Error('Please provide path and extension');
```

Comment:
if we throw error without try/catch, the app will crash

Reply:
ok I remove it